### ISSUE TITLE
How to troubleshoot issues ?

### ISSUE THREAD

**Issue by corentincharron** [docker-taiga/taiga#1]
Hello !
Thanks a lot for the images, they seem to really clean. Although, I integrated taiga-back, taiga-front, taiga-db, taiga-events and taiga-rabbit in my existing docker-compose file (as I already have other services running, such as gitlab and a nginx reverse proxy, which is why I didn't keep the original taiga-proxy from your docker-compose file.

After running it, I get to the Taiga home page, but it says: _Something happened and our Oompa Loompas are working on it._

Where can I find the logs that would help me to understand what I did wrong ?

Cheers.

**Comment by w1ck3dg0ph3r** [comment]
Hi! I would check browser console and XHR requests log first, then the logs of taiga-back container via `docker logs`. Could it be something with the nginx config?

**Comment by corentincharron** [comment]
Ho, I did not think about the browser console! Good shout, thanks a lot!

So I implemented the nginx.conf from the proxy container into mine, which helped considerably, and now I can't log in with admin/123123.

According to the Network tab of the Inspector, I'm getting a 502 when I try to log in (POST, auth, right ?). The console doesn't say anything... Does that mean that I wrongly copied the nginx configuration?

**EDIT**: It actually gets a 502 when loading the discover page, it looks like it can't find (or access ?) the API at all, even if all the url in the settings are right.
